BiweeklyMeetings/2022-05-24.md

```diff
@@ -0,0 +1,563 @@
+
+**Sebastien Guillemot**:  So the problem with this proposal is that, they're basically saying, "Oh yeah, you'll use the same policy for... And you made it a 808." This doesn't help existing projects, and doesn't describe what happens if somebody overrides their existing media in the future. But this is more of a Cardano issue than an issue with a specific proposal. And I'm not sure if there's anything that they can do with this.
+
+**Sebastien Guillemot**:  Since this proposal is isolated to a new metadata tag, a new label, it's not overlapping with anybody else's proposals. So I don't think frankly speaking, there's a reason to reject it, given that it... There's no kind of overlap with other people that have, so this isn't stop people from making competing standards. I think the proposal itself could be refined a bit. Right now it's kind of unnecessarily long for Twitter links and some other stuff that probably should not go into the CIP itself. So we are going to do this kind of material cleanups.
+
+**Sebastien Guillemot**:  It'd be good to know if anybody intends to actually implement this specific standard. So we could just say, okay, well, there's nothing wrong with this standard, headway speaking. And it's a fairly simple one too. So we could merge it, but it'd be good to know if there's at least one marketplace that actually is interested in this idea and to see what they think, or if this is just a one-time standard. Do you know if any marketplace has commented on this issue? I don't think so. I think all these links are just people who think it would be a good idea to have something like this.
+
+**M. Àngels Jover**:  I mean, we have comments from Matthias, Robert.
+
+**Matthias Benkort**: No, it has been showing a few tweets. I'm not quite sure. I haven't checked, so I'm not sure what the marketplace think of it. But yeah, generally speaking, I do agree with Sebastien, that there is no particular conflict in merging that as a pro CIP, as long as maybe we have a clear path to active, that is perhaps one marketplace or at least one actor, which is interested in adopting it, and something that can be then reflected upon.
+
+**M. Àngels Jover**:  And are we going to reach out the marketplace in order to review this, or how do you want to proceed?
+
+**Matthias Benkort**: So, we could put it as a last check for next time maybe, and comment in the meantime to make sure that the author adds a clear path to active section mentioning... Yeah, so there is probably one marketplace that is going on with that CIP. And also just to have a... Two weeks period of last check and potentially hear from people that if we have it brought to the light.
+
+**Discord discussion on the PR**
```

Comment:
```suggestion
##### Discord discussion on the PR
```
There are 4 more of these, to change to a subheading to match the last change (https://github.com/cardano-foundation/CIPs/commit/baf6c52336623f77efb0f366b788e44766e4ac7c)... including this one 😁